src/devices/Units/tests/PressureTests.cs

```diff
@@ -47,7 +47,7 @@ public void ToStringDefault()
             using (new SetCultureForTest("de-CH"))
             {
                 Pressure p = Pressure.FromHectopascal(1024.24);
-                Assert.Equal("1024.24hPa", p.ToString());
+                Assert.Equal("1024.2 hPa", p.ToString());
```

Comment:
please add at least 3 tests:
- ToString with just units no number of decimal places
- ToString with single digit number of decimal places
- ToString with double digit number of decimal places (if that makes sense)